templates/injector-network-policy.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (eq (.Values.injector.enabled | toString) "true" ) (and (eq (.Values.global.enabled | toString) "true") (eq (.Values.global.openshift | toString) "true") ) }}
+{{- if eq (.Values.injector.networkPolicy.enabled | toString) "true" }}
```

Comment:
I believe this would create a networkPolicy even if you disabled the injector, which shouldn't happen.